src/components/CustomInputs/Toggle.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Toggle = ({ labelOn, labelOff }) => {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const handleCheck = (e) => {
```

Comment:
I thought that it would be sort of "more reliable" if I got the actual value from the input directly. As if, for some reason, the value of the checkbox gets updated without having called the onChange event just once, or something...

Reply:
Check these articles:

https://reactjs.org/docs/forms.html#controlled-components
https://reactjs.org/docs/uncontrolled-components.html

TL;DR: Common practice is to make your form inputs be controlled with React state.